#agents' loc, player's loc, lcookies'loc, scookies'loc ...
custom_msgs/axis[] player_axis
custom_msgs/axis[] agents_axis
custom_msgs/axis[] lcookies_loc
custom_msgs/axis[] scookies_loc
custom_msgs/axis[] defa
custom_msgs/axis[][] map
bool[] ghost
bool selecting_menu
int32 score
int32 life

